src/bsp/pc-rtems/build_options.cmake

```diff
@@ -5,6 +5,13 @@
 ##########################################################################
 
 # Link the RTEMS BSP with the "rtemscpu" system library
-target_link_libraries(osal_bsp
-    rtemscpu
-)
+if(CMAKE_SYSTEM_VERSION GREATER 5)
+    target_link_libraries(osal_bsp
+         rtemscpu
+         networking
+    )
+else()
+    target_link_libraries(osal_bsp
+          rtemscpu
+    )
+endif(CMAKE_SYSTEM_VERSION GREATER 5)
```

Comment:
Just a minor/pedantic comment that "target_link_libraries" is defined by CMake to append libraries to the link list, it does not replace or change existing link libraries.  Therefore this could be simplified, where "rtemscpu" is always added first (as it was) and then just add the RTEMS 6 specific library like so:

```
if(CMAKE_SYSTEM_VERSION GREATER 5)
    target_link_libraries(osal_bsp networking)
endif()
```

This would also simplify things a bit if someday we want to build an RTEMS 6 image without networking - because one would still need rtemscpu in that case too.

Reply:
Good suggestion, I'll make that change.